### DISCUSSION TITLE
Verify two factors authentication

### DISCUSSION THREAD

**Discussion by djedjethai** [community/community#67100] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I don't know if it's the right place to ask that, but I don't see any other section matching my question...

For the past 2-3 days, every time I attempt to access my account via the browser, I am directed to a page that prompts me to confirm 2FA. The message instructs me to open my GitHub mobile app to obtain the code (TOTP). However, I am not receiving any code in the app. Interestingly, 2FA does work, for instance I get the code when I delete a repository.

I like to activate the sms 2FA as well, but unfortunatly I can not since I reside in Thailand, and GitHub does not support Thai phone numbers.

Did I miss something ??

Thanks for any advices.

**Comment by patrykkawiak** [comment]
You can use your password to login instead code. Then try to disable 2FA and enable it again

**Comment by djedjethai** [comment]
Hi, thanks for the answer. I disabled it but now I can not enable it anymore. Got the code(refreshing every few seconds as expected) on the TOTP app but they are invalid...

**Comment by djedjethai** [comment]
ok it works, I had to remove my Github account from the TOTP app then re-add it again. Thanks @patrykkawiak
